packages/jaeger-ui-components/src/TracePageHeader/TracePageHeader.test.tsx

```diff
@@ -14,25 +14,34 @@
 
 import { render, screen } from '@testing-library/react';
 import React from 'react';
+import { ViewRangeTimeUpdate } from 'src/TraceTimelineViewer/types';
 
 import traceGenerator from '../demo/trace-generators';
 import { getTraceName } from '../model/trace-viewer';
 import transformTraceData from '../model/transform-trace-data';
 
-import TracePageHeader from './TracePageHeader';
+import TracePageHeader, { TracePageHeaderEmbedProps } from './TracePageHeader';
 
 const trace = transformTraceData(traceGenerator.trace({}));
-const setup = (propOverrides) => {
+const setup = (propOverrides?: TracePageHeaderEmbedProps) => {
   const defaultProps = {
+    canCollapse: false,
+    hideSummary: false,
+    onSlimViewClicked: () => {},
+    onTraceGraphViewClicked: () => {},
+    slimView: false,
     trace,
     hideMap: false,
     showArchiveButton: false,
     showShortcutsHelp: false,
     showStandaloneLink: false,
     showViewOptions: false,
     textFilter: '',
```

Comment:
Could you remove these? Doesn't look like they're used anywhere :)

Reply:
https://github.com/grafana/grafana/pull/59256/commits/98a9328f3a3b9c76b299bb2b0d5348d5bc324097